Tasks/VsTestV2/task.json

```diff
@@ -535,6 +535,7 @@
         "configureDtaAgentFailed": "Configuring the test agent with the server failed even after %d retries with error %s",
         "otherConsoleOptionsNotSupported": "Other console options is not supported when using the multi-agent phase setting. This option will be ignored.",
         "distributedTestWorkflow": "In distributed testing flow",
+        "nonDistributedTestWorkflow": "In single agent testing flow",
```

Comment:
this will confuse customers. we shouldn't use 'Agents' in our description.
'Running tests using vstest.console.exe runner?'

Reply:
Ok